WooCommerce/Classes/ViewRelated/Orders/OrdersViewController.swift

```diff
@@ -584,7 +569,12 @@ extension OrdersViewController: UITableViewDelegate {
             return
         }
 
-        performSegue(withIdentifier: Segues.orderDetails, sender: detailsViewModel(at: indexPath))
+        guard let orderDetailsVC = OrderDetailsViewController.instantiatedViewControllerFromStoryboard() else {
+            return
```

Comment:
nit: not sure if it's worth adding an `assertionFailure` before the return?

Reply:
I like it! Done in 6094c2cf0e91ef66501e8c9de8eb229975bdd8da.